resources/web/panel/js/pages/keywords/emotesAndEmoji.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2016-2021 phantombot.github.io/PhantomBot
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+page = {
```

Comment:
Global variable declared, enclose local scoped code inside lambda function and explicitly export only as necessary
```
$(function () {
    //Code here
});
```